README.md

```diff
@@ -1,6 +1,12 @@
 # kruize-ui
 This repository contains the code for Kruize Autotune user interface. The objective behind creating the UI is to enable newbies to explore and contribute to Autotune and simplify the initial steps of starting an Autotune experiment. The UI significantly reduces the burden on user to understand, operate and work with Autotune. Not only this it also helps better explain and present what remarkable optimizations Autotune can bring to site reliability engineering.
 
+## Quick Start
+git clone https://github.com/kruize/kruize-ui.git <br />
+cd kruize-ui <br />
+npm install @patternfly/react-core --save <br />
+npm install && npm run start:dev
+
```

Comment:
@bhanvimenghani Can you please add a `launch.sh` bash script that will do all of the above and bring up the UI?

Reply:
yes, sure!